### PR TITLE
[feat] MB-1192: Add not passing, course ended status to certificate status alert

### PR DESCRIPTION
**Pull request by Tj-Tracy**
image

mobile:
image

### CODE REVIEW

File: src/course-home/outline-tab/alerts/certificate-status-alert/CertificateStatusAlert.jsx [hunk 1]
```diff
@@ -118,6 +120,24 @@ function CertificateStatusAlert({ intl, payload }) {
     return alertProps;
   };
 
+  const renderNotPassingCourseEneded = () => {
```
Comment (oliviaruizknott) [inline src/course-home/outline-tab/alerts/certificate-status-alert/CertificateStatusAlert.jsx]: Typo? `Eneded` -> `Ended`

Comment (Tj-Tracy) [reply src/course-home/outline-tab/alerts/certificate-status-alert/CertificateStatusAlert.jsx]: fiexed :)

File: src/course-home/data/__snapshots__/redux.test.js.snap [hunk 2]
```diff
@@ -461,6 +461,7 @@ Object {
           "canEnroll": true,
           "extraText": "Contact the administrator.",
         },
+        "enrollmentMode": undefined,
```
Comment (justinhynes) [inline src/course-home/data/__snapshots__/redux.test.js.snap:464]: As someone who still isn't comfortable/proficient with Javascript... Is there any reason some things are defined with a default of `undefined` vs. `null`? (I realize this looks to be test data, I was curious)

**Review by justinhynes — APPROVED** [review]
LGTM!

File: src/course-home/data/__snapshots__/redux.test.js.snap (hunk 2 above)
Comment (Tj-Tracy) [reply src/course-home/data/__snapshots__/redux.test.js.snap:464]: the short of it is: `null` is an assigned state of the absence of value. Its usually used to explicitly state "this is deemed to have no value by some code". Where as, `undefined` is the default state given by javascript to things that implicitly have no value. But like everything else in JS, non of this is really that strict, and they basically mean the same thing.

Comment (justinhynes) [reply src/course-home/data/__snapshots__/redux.test.js.snap:464]: Thanks!

**Pull request merged**